### PR TITLE
Dockerization of frontend app + cdn

### PR DESCRIPTION
**Pull request by mateuszc-kainos**
https://companieshouse.atlassian.net/browse/BI-3666

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+WORKDIR /build
+COPY package.json package-lock.json ./
+
+## Image with runtime dependencies
+FROM build-base as prod-deps-image
+
+RUN npm install --production
+
+## Build image
```
Comment (sapphireduffy) [inline Dockerfile:15]: Do we need to keep these comments in?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      - COOKIE_NAME=SID
+      - COOKIE_DOMAIN=chs-dev.internal
+      - COOKIE_SECURE_ONLY=0
+      - COOKIE_SECRET=ChGovUk-XQrbf3sLj2abFxIY2TlapsJ
+      - DEFAULT_SESSION_EXPIRATION=3600
+
+      - ACCOUNT_WEB_URL=http://web.chs-dev.internal:4000
+
+      # - FILE_TRANSFER_API_URL:
+      # - FILE_TRANSFER_API_KEY:
```
Comment (sapphireduffy) [inline docker-compose.yml:40]: Can we remove these comments? Or do we need to keep them?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    networks:
+      - chs-dev
+    ports:
+      - 80:80
+      - 8080:8080 # Web UI (enabled by --api.insecure=true)
+  dissolution-web:
+    build: .
+    image: dissolution-web:1.0
+    labels:
+      - traefik.http.routers.lfp-appeals-frontend.rule=Host(`chs-dev`) && PathPrefix(`/dissolution-web/`)
```
Comment (peadarkelly) [inline docker-compose.yml:19]: appeals reference needs changed

File: docker-compose.yml (hunk 2 above)
Comment (peadarkelly) [reply docker-compose.yml:40]: I agree, let's only keep the variables we need

File: traefik.docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+version: "3.8"
```
Comment (peadarkelly) [inline traefik.docker-compose.yaml:1]: Is this file needed? This block appears in the docker-compose.yml already

Comment (mateuszc-kainos) [reply traefik.docker-compose.yaml:1]: It is needed, docker won't run without it.
